joint-strategy-subcommittee/2018-04-04.md

```diff
@@ -0,0 +1,107 @@
+## Node.js Board Strategy Subcommittee
+
+April 4, 2018
+
+#### Present: 
+
+- Mark Hinkle, Executive Director
+- Todd Moore, Chair, Node.js Board of Directors
+- Sarah Novotny, Subcommittee Co-Chair, Node.js Board of Directors
+- Brian McCallister, Node.js Board of Directors
+- Ashley Williams, Node.js Board of Directors
+- Michael Dawson, TSC Chair
+- Tierney Cyren, CommComm Chair
+- Myles Borin, TSC Director
+- Tracy Hinds, Subcommittee Co-Chair, CommComm member
```

Comment:
Oops! I didn't make the list.

Reply:
Ah I will add! I remember you being there now but I think it was after we'd gotten going and added the list of attendees.